ui/values/localizable/en.go

```diff
@@ -790,4 +790,7 @@ const EN = `
 "assets" = "Assets"
 "noWalletsAvailable" = "You cannot spend from a watch only wallet, try creating another wallet."
 "createAssetWalletToSwapMsg" = "You need to create a %s wallet to swap."
+"stakingInfo" = "Staking Info"
+"timeLeftTitle" = "Time Left"
```

Comment:
Please rename the other `timeLeft` to `timeLeftFmt` and rename this to `timeLeft`.